mam/v2/mam/mam.h

```diff
@@ -119,16 +119,16 @@ mam_endpoint_save, mam_endpoint_load
 #define MAM2_PSK_ID_SIZE 81
 #define MAM2_PSK_SIZE 243
 /*! \brief Preshared key. */
-typedef struct mam_psk_s {
+typedef struct mam_pre_shared_key_s {
   word_t id[MAM2_WORDS(MAM2_PSK_ID_SIZE)];
-  word_t psk[MAM2_WORDS(MAM2_PSK_SIZE)];
-} mam_psk_t;
+  word_t pre_shared_key[MAM2_WORDS(MAM2_PSK_SIZE)];
+} mam_pre_shared_key_t;
 
-trits_t mam_psk_id(mam_psk_t *p);
-trits_t mam_psk_trits(mam_psk_t *p);
+trits_t mam_psk_id(mam_pre_shared_key_t *p);
+trits_t mam_psk_trits(mam_pre_shared_key_t *p);
 
-def_mam_list_node(mam_psk_t, mam_psk_node);
-def_mam_list(mam_psk_node, mam_psk_list);
+def_mam_list_node(mam_pre_shared_key_t, mam_pre_shared_key_node);
+def_mam_list(mam_pre_shared_key_node, mam_pre_shared_keys_list);
```

Comment:
I know they weren't, but can you suffix nodes and lists types with `_t` for consistency ?

Reply:
sure